src/core/chatArgsValidator.spec.js

```diff
@@ -1,7 +1,7 @@
 import Utils from "../utils";
 import { ChatServiceArgsValidator } from "./chatArgsValidator";
 import { IllegalJsonException } from "./exceptions";
-
+import regeneratorRuntime from "regenerator-runtime";
```

Comment:
Use [jest's `setupFiles` config](https://jestjs.io/docs/en/configuration#setupfiles-array) to always import this so we don't need to do this in every spec file.

Reply:
Nice. Good find. I'll update the cr.